app/forms/idv/api_image_upload_form.rb

```diff
@@ -219,13 +219,19 @@ def update_analytics(client_response)
         ).merge(native_camera_ab_test_data),
       )
       pii_from_doc = client_response.pii_from_doc || {}
-      store_encrypted_images_if_required
+      stored_image_result = store_encrypted_images_if_required
       irs_attempts_api_tracker.idv_document_upload_submitted(
         success: client_response.success?,
         document_state: pii_from_doc[:state],
         document_number: pii_from_doc[:state_id_number],
         document_issued: pii_from_doc[:state_id_issued],
         document_expiration: pii_from_doc[:state_id_expiration],
+        front_image: stored_image_result&.front_uuid,
+        front_image_content_type: (front.content_type if stored_image_result),
+        front_image_encryption_key: stored_image_result&.front_encryption_key,
+        back_image: stored_image_result&.back_uuid,
+        back_image_content_type: (back.content_type if stored_image_result),
```

Comment:
unsure it's clearer to other people but the trailing `if` syntax was surpising to me. Two ideas:

first, just `&&`

```suggestion
        front_image_content_type: stored_image_result && front.content_type,
        front_image_encryption_key: stored_image_result&.front_encryption_key,
        back_image: stored_image_result&.back_uuid,
        back_image_content_type: stored_image_result && back.content_type,
```

second, a ternary:

```suggestion
        front_image_content_type: stored_image_result ? front.content_type : nil,
        front_image_encryption_key: stored_image_result&.front_encryption_key,
        back_image: stored_image_result&.back_uuid,
        back_image_content_type: stored_image_result ? back.content_type : nil,
```

Reply:
Thanks! Definitely looking for feedback on what appropriate Rubyisms are